.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+const { rules } = require('eslint-config-jitsi');
```

Comment:
is this the same as in the other TS repo? maybe we can share this config in the eslint repo?

Reply:
Or maybe we should start creating an eslint set of rules for TS. I think they are different enough to do it. Maybe an R&D day?

Reply:
Sounds good!